code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -142,6 +142,11 @@
 	toxpwr = 2
 	taste_description = "fish"
 
+/datum/reagent/toxin/carpotoxin/on_mob_life(mob/living/carbon/M)
```

Comment:
No need to keep updating constantly on mob life, do it on mob metabolise and it's only done once then

Reply:
alright lets see if I can fiugur out how to make it do that i think I got it but lets SEEyes

Reply:
I think that did it??!??!?!?!?!????????